examples/sycl_interop_usm.cpp

```diff
@@ -31,23 +31,23 @@ class kernel_tag;
 
 void sycl_usm_tutorial(engine::kind engine_kind) {
 
-    engine eng(engine_kind, 0);
```

Comment:
I think we better stick to the old version of the example because using `gpu_selector` doesn't guarantee that the device will the Intel's one. Inside the library we filter out all non-intel devices hence it's safe to use `engine eng(engine_kind, 0);`

Reply:
good idea.